public/components/visualization/text2viz.tsx

```diff
@@ -243,6 +268,15 @@ export const Text2Viz = () => {
             }),
           });
           dialog.close();
+
+          // Report metric when a new visualization is saved.
+          if (usageCollection) {
+            usageCollection.reportUiStats(
+              VIS_NLQ_APP_ID,
+              usageCollection.METRIC_TYPE.LOADED,
+              `saved-${uuidv4()}`
+            );
```

Comment:
out of courisity, how we are going to use those metrics? For example, how to get the statics of how many user save the visalization?

Reply:
I guess we don't need it any more? @chishui any idea?

Reply:
> I guess we don't need it any more?

You mean uuid or metric itself? We need both